generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -5,12 +5,16 @@
     kill_vm_on_error = yes
     login_timeout = 240
     reboot_count = 25
-    slots_mem = 4
-    maxmem_mem = 32G
     setup_hugepages = yes
-    mem_devs = hugepage
-    size_mem_hugepage = 256M
-    backend_mem = memory-backend-file
```

Comment:
I put the previous scenario in boot_vm_in_hugepage.2M_hugepage and changed the parameters according to feature owner's advice. Thanks!